src/python/pants/backend/python/dependency_inference/module_mapper_test.py

```diff
@@ -197,26 +190,17 @@ def test_map_first_party_modules_to_addresses(rule_runner: RuleRunner) -> None:
             "src/python/project/util/dirutil.py": "",
```

Comment:
This function was simplified - there's no concept of "ambiguity" with this codepath anymore, so fewer edge cases to test.